docs/production-deployment/cloud/api-keys.mdx

```diff
@@ -314,6 +314,12 @@ Follow these steps to rotate API keys:
 1. Switch clients to load the new key and start using it.
 1. Delete the old key after it is no longer in use.
 
+:::tip
+
+Service accounts have the permission to rotate their own API keys. To use this feature, have your service account create a new API key using the [Cloud Ops APIs](/ops) before the current one expires. Note that service accounts cannot delete their own API keys, which helps keep workflow access secure.
```

Comment:
may be worth mentioning that the initial key must be provisioned by an account admin (or higher).

Reply:
i updated to note that you don't need to configure "permissions" in order to rotate api keys. kept it a little open ended in cases we change the terminology of admin roles